actgen/wrappers/duplicate_actions.py

```diff
@@ -0,0 +1,40 @@
+import gym
+
+
+class DuplicateActions(gym.Wrapper):
+    """
+    duplicate the original actions in a gym environment
+    """
+    def __init__(self, env, n_dup):
+        """
+        :param env: an unwrapped gym environment that has discrete actions
+        :param n_dup: the number of times to duplicate the original actions
+                        e.g. n_dup = 3 in a cartpole environment will result in a total of 6 actions
+                            in the order of (L1, R1, L2, R2, L3, R3)
+        """
+        super(DuplicateActions, self).__init__(env)
+        self.n_dup = n_dup
+        self.num_actions = n_dup * env.action_space.n
+        self.action_space = gym.spaces.Discrete(self.num_actions)
+        self.observation_space = env.observation_space
+        self.env = env
+        self._max_episode_steps = env._max_episode_steps
```

Comment:
Same for this. The purpose of the FixedDurationHack wrapper is to avoid needing to do this with every subsequent wrapper.